boms/common-ee/pom.xml

```diff
@@ -771,6 +771,13 @@
                 <version>${version.com.fasterxml.jackson.jr.jackson-jr-objects}</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.codehaus.plexus</groupId>
+                <artifactId>plexus-utils</artifactId>
+                <version>${version.org.codehaus.plexus.plexus-utils}</version>
+                <scope>test</scope>
+            </dependency>
```

Comment:
Please move this into the root pom. This bom should only include production code dependencies. I'd like to create a testbom in full WF like we have in WF Core[1], so until we do that I'm leaving test deps in the root pom and add comments to them to note they are testsuite-only.

[1] https://issues.redhat.com/browse/WFLY-17412

Reply:
fixed, I read from pom.xml there is https://issues.redhat.com/browse/WFLY-17331 already.